apps/src/templates/lessonOverview/lessonPlanShapes.jsx

```diff
@@ -91,3 +91,24 @@ export const lessonShape = PropTypes.shape({
   objectives: PropTypes.arrayOf(PropTypes.object).isRequired,
   assessmentOpportunities: PropTypes.string
 });
+
+export const reducedLessonShape = PropTypes.shape({
```

Comment:
I'm wondering if we want to call this studentLessonShape and then possibly even create a smaller shape for the navigation dropdown with only the critical parts called something like navigationLessonShape (I probably should have done that originally)

Reply:
That sounds like a great idea! I'll add that